MAUI/ImageEditor/shape.md

```diff
@@ -0,0 +1,206 @@
+---
+layout: post
+title: Shape Annotation in .NET MAUI ImageEditor control | Syncfusion
+description: Learn about the shape annotation feature of Syncfusion .NET MAUI ImageEditor(SfImageEditor) control.
+platform: .NET MAUI
+control: SfImageEditor
+documentation: ug
+---
+
+# Shape Annotation in .NET MAUI ImageEditor (SfImageEditor)
```

Comment:
Shape annotations in .NET MAUI Image Editor (SfImageEditor)

Reply:
Changed a heading